controllers/dataController.js

```diff
@@ -0,0 +1,74 @@
+const axios = require('axios');
+const HttpError = require('../models/httpError');
+const { createURL } = require('../utils/helper');
+
+const getAll = (req, res, next) => {
```

Comment:
Technically we only need this controller action. As discussed, believe me duplicating these will not make our life easier.